### PR TITLE
docs(configuration options): Add a CodePen for validating whether a later.js schedule is valid or not.

### PR DESCRIPTION
**Pull request by paulzakin**
## Changes

- Add a CodePen for validating whether a later.js schedule is valid or not.

## Context

I think for individuals who aren't familiar with later.js (like me!), it can be difficult to tell whether a `schedule` is valid. Plus, I found their documentation was missing some sort of "demo" feature. Hence, I made a quick CodePen to validate whether the proposed schedule is valid. It also lists the next two occurences in that schedule. I found this very helpful in reaching an intuitive understanding of how the schedules worked, and it may be helpful to other users as well.

> Happy to transfer/surrender ownership of the CodePen to renovate as needed. 

## Documentation (please check one with an [x])

- [X] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [X] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by rarkins** [comment]
One challenge here - Renovate accepts only a *subset* of Later.js schedules. For example the default "every 5 minutes" is not a valid Renovate schedule. Can this codepen be made to mimic what Renovate considers valid?

**Comment by paulzakin** [comment]
@rarkins No promises but happy to take a look at modifying it! 

Any pointers on where in the codebase should I look for that Renovate-flavored Later.js? Happy to look myself, but appreciate any points in the right direction!

**Comment by rarkins** [comment]
The logic is all in https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/update/branch/schedule.ts

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Comment by paulzakin** [comment]
> The logic is all in https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/update/branch/schedule.ts

Awesome will take a look

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -2649,6 +2649,11 @@ Renovate does not support scheduled minutes or "at an exact time" granularity.
 !!! note
     Actions triggered via the [Dependency Dashboard](https://docs.renovatebot.com/configuration-options/#dependencydashboard) are not restricted by a configured schedule.
 
+<!-- prettier-ignore -->
+!!! tip
+    To validate your `later` schedule before updating your `renovate.json`, you can use [this CodePen](https://codepen.io/rationaltiger24/full/ZExQEgK).
+
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:2655]:
```suggestion
```

Drop this double newline. 😄

**Comment by paulzakin** [comment]
Alright, updated with the rules - take a look:

https://codepen.io/rationaltiger24/pen/ZExQEgK

**Comment by rarkins** [comment]
> By I assume Rennovate would not run every minute under this configuration? Or am I missing something?

That's right. Schedules do not *trigger* Renovate to run. Instead, if Renovate is running AND the schedule matches at that time, then it will create branches/PRs.

**Comment by paulzakin** [comment]
> > By I assume Rennovate would not run every minute under this configuration? Or am I missing something?
> 
> That's right. Schedules do not _trigger_ Renovate to run. Instead, if Renovate is running AND the schedule matches at that time, then it will create branches/PRs.

That makes sense, but what triggers Renovate to run, assuming it is a GitHub app that is installed

**Comment by viceice** [comment]
> > > By I assume Rennovate would not run every minute under this configuration? Or am I missing something?
> > 
> > 
> > That's right. Schedules do not _trigger_ Renovate to run. Instead, if Renovate is running AND the schedule matches at that time, then it will create branches/PRs.
> 
> That makes sense, but what triggers Renovate to run, assuming it is a GitHub app that is installed

It tries to run hourly ba default and reduces cycles if no activity on repo. You can always trigger a run from dependency dashboard.

**Comment by paulzakin** [comment]
> > > > By I assume Rennovate would not run every minute under this configuration? Or am I missing something?
> > > 
> > > 
> > > That's right. Schedules do not _trigger_ Renovate to run. Instead, if Renovate is running AND the schedule matches at that time, then it will create branches/PRs.
> > 
> > 
> > That makes sense, but what triggers Renovate to run, assuming it is a GitHub app that is installed
> 
> It tries to run hourly by default and reduces cycles if no activity on repo. You can always trigger a run from dependency dashboard.

Perfect that makes sense to me!

**Review by rarkins — APPROVED** [review]

**Pull request merged**